### PR TITLE
fix #1877, Several navigation issues on Mobile phone

### LINKED ISSUE #1877: Several navigation issues on Mobile phone
Current problems:

1. Modal dialog popup is too big to be displayed. And after clicking, user cannot cancel it as cancel button is hidden. 
2. No right-click operation support.
3. Uptime, Application Click, and Processor text overlap with each other.

### PR DESCRIPTION
**Pull request by clockfly**
Current problems:

1. Modal dialog popup is too big to be displayed. And after clicking, user cannot cancel it as cancel button is hidden. 
2. No right-click operation support.
3. Uptime, Application Click, and Processor widget overlap with each other.

### CODE REVIEW

File: services/dashboard/styles/bootstrap-override.css [hunk 1]
```diff
@@ -155,7 +155,7 @@ html, body {
 
 /* Specify the size of modal dialog */
 .modal-md {
-  width: 520px;
+  margin-top: 30px;
```
Comment (stanleyxu2005) [inline services/dashboard/styles/bootstrap-override.css]: Why margin top? Please update the comment above.

File: services/dashboard/views/apps/compose/popups/choose_edge.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="modal" tabindex="-1">
-  <div class="modal-dialog modal-md">
+  <div class="modal-md col-md-5 col-md-offset-3">
```
Comment (stanleyxu2005) [inline services/dashboard/views/apps/compose/popups/choose_edge.html]: `col-md-5 col-md-offset-3` Do you want to a center alignment? Please try `modal-md text-center`

File: services/dashboard/views/apps/streamingapp/streamingapp.html [hunk 3]
```diff
@@ -8,13 +8,13 @@
       </h4>
       <h5 ng-bind="app.description"></h5>
     </div>
-    <div class="col-md-2 col-sm-3 col-xs-4">
+    <div class="col-md-2 col-sm-3 col-xs-3">
```
Comment (stanleyxu2005) [inline services/dashboard/views/apps/streamingapp/streamingapp.html]: Don't change that, `col-xs-4` is intended to align property table below

File: services/dashboard/views/apps/compose/popups/choose_edge.html (hunk 2 above)
Comment (clockfly) [inline services/dashboard/views/apps/compose/popups/choose_edge.html]: Tried, "text-center" looks not good.
modal

File: services/dashboard/styles/bootstrap-override.css (hunk 1 above)
Comment (clockfly) [inline services/dashboard/styles/bootstrap-override.css]: The margin is consistent with bootstrap modal dialog style.
Check:
margin

File: services/dashboard/views/apps/streamingapp/streamingapp.html (hunk 3 above)
Comment (clockfly) [inline services/dashboard/views/apps/streamingapp/streamingapp.html]: I didn't find there is any alignment issue. Here is the screen shot for iPhone 4.
align

Comment (clockfly) [inline services/dashboard/views/apps/streamingapp/streamingapp.html]: Can you add a screenshot to describe the problems?

Comment (stanleyxu2005) [inline services/dashboard/views/apps/streamingapp/streamingapp.html]: "Application clock" is expected to align with the "akka.tcp://...".

In general, i would suggest break "2 processors 3 executors" in next line. As result: "col-xs-4", "col-xs-8", "col-xs-12"

File: services/dashboard/views/apps/compose/popups/choose_edge.html (hunk 2 above)
Comment (stanleyxu2005) [inline services/dashboard/views/apps/compose/popups/choose_edge.html]: `modal-md` is my invented style, You still need `modal` as base style. Don't remove that.

Comment (clockfly) [inline services/dashboard/views/apps/compose/popups/choose_edge.html]: Please add a screenshot if you think anything is wrong.

File: services/dashboard/views/apps/streamingapp/streamingapp.html (hunk 3 above)
Comment (clockfly) [inline services/dashboard/views/apps/streamingapp/streamingapp.html]: Why "Application Clock" need to align with actorPath field of another table? As they are two different tables, it looks to me not that necessary to do the align.

I tried the "col-xs-4", "col-xs-8", "col-xs-12" you suggested. Here is the screenshot after the change. If you think this looks better, I can update the patch.
align12

Comment (clockfly) [inline services/dashboard/views/apps/streamingapp/streamingapp.html]: For "col-xs-4", "col-xs-8", "col-xs-12", this is how it looks for nexus 7.
nexux_7

Comment (stanleyxu2005) [inline services/dashboard/views/apps/streamingapp/streamingapp.html]: Yes, this is my preference. All other pages follow this align concept.

Comment (clockfly) [inline services/dashboard/views/apps/streamingapp/streamingapp.html]: Ok, updated.

File: services/dashboard/views/apps/compose/popups/choose_edge.html (hunk 2 above)
Comment (stanleyxu2005) [inline services/dashboard/views/apps/compose/popups/choose_edge.html]: +1

**Pull request merged**